### DISCUSSION TITLE
Would it be possible add "go to specific date" button or command?

### DISCUSSION THREAD

**Discussion by wm23** [davish/obsidian-full-calendar#444] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Now when I want to see day from the past in example 3 years ago I need click "<-" button 36 times or even worse  on the mobile "3 days view".

### ✔️ Solution

Maybe it could be possible to add button "go to date" or command "Full Calendar - open specific date"?

**Comment by milk717** [comment]
I want this feature very much.
I've looked up both issues and discussions, but I don't think this feature has been added yet.
Is there any other way to make this function work similarly?
